restaurant/src/App.jsx

```diff
@@ -0,0 +1,32 @@
+import "./App.css";
+import Home from "./assets/home";
```

Comment:
Assets folder is supposed to be for images, not for source code. You can move your other code components in a folder called `components` or similar